api/docker-compose.yaml

```diff
@@ -96,9 +96,28 @@ services:
       volumes:
         - gradle-cache:/.gradle
         - .:/w:cached
+        - ~/.config:/.config
       command: wait-for db:3306 -- ./run-bigdata-migrations.sh bigdata local
       env_file:
         - db-cdr/vars.env
+  db-generate-cloudsql-cdr:
```

Comment:
Are you going to add docker or Ruby commands for the other scripts?

Reply:
??? Just goal is run command i sent more or less  to do what we want.